internal/stage/filebased/filebased.go

```diff
@@ -0,0 +1,39 @@
+package filebased
+
+import (
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/redhatinsights/insights-ingress-go/internal/stage"
+)
+
+// Stager provides the mechanism to stage a payload to the file system
+type Stager struct {
+	StagePath string
+	BaseURL   string
+}
```

Comment:
Can we rename this to FilebasedStager or something like that?  Its confusing to have a concrete implementation named the same as the Interface.

Reply:
Sure. Should I update the S3 one, as its named this way as well?

Reply:
Yeah, if you don't mind too, that would be great!

Reply:
done